doc/man7/openssl-core.h.pod

```diff
@@ -10,7 +10,7 @@ openssl/core.h - OpenSSL Core types
 
 =head1 DESCRIPTION
 
-The <openssl/core.h> header file defines a number of public types that
+The header file C<openssl/core.h> defines a number of public types that
```

Comment:
Wellllllll....  unmarked `<whatever>` is rendered exactly like that (see for yourself in https://www.openssl.org/docs/manmaster/man7/openssl-core.h.html), and *that is correct*, as per man-pages(7):

> **Formatting conventions (general)**
> ... When referring to a standard header file include, specify the header file surrounded by angle brackets, in the usual C way (e.g., \<stdio.h\>). ...

However, in that same piece of text, there's one thing we're not doing well yet:

> Filenames (whether pathnames, or references to header files) are always in italics (e.g., *\<stdio.h\>*)

However, the remedy is quite easy, there's the `F<>`, which does render into italics.  However, to keep the angle brackets, we need to have this: `F<< <openssl/core.h> >>`

Side note: it is mentioned in [perlpodstyle](https://perldoc.perl.org/perlpodstyle.html) that all files names should be enclosed in `F<>`...  this aligns pretty well with man-page conventions.
(I would take exception for file names that are part of terminal input or output, though)

If we do that, we also do not need to say that it's a header file, the sentence could simply start like this:

```
F<< <openssl/core.h> >> defines a number ...
```

Reply:
Ping

Reply:
I am convinced that GitHub is gaslighting me and that it hides comments just to come back and go BOO later on. :)  Fixed.  I started with "The file F<< ..." hope that's okay.